applications/plugins/CGALPlugin/src/CGALPlugin/Refine2DMesh.inl

```diff
@@ -351,7 +351,7 @@ namespace cgal
 				}
 			}
 		}
-		
+#endif
```

Comment:
Disabling code for everybody because it is not compiling with CGAL5  is not really nice 🧨 (CGAL 4.X guys wont appreciate)
If not fixing, then at least enclose the disabled code with check on CGAL version + a pragma message telling that Refine2DMesh is not working anymore with CGAL5

Reply:
Of course ! What is the macro ? 
According to me this file can be completely removed with cgal 5 because the component will exists but with an incorrect behavior. Who need this component ?

Reply:
I fixed this code for CGAL > 5, but I cannot push in this branch.